cli/cmd/common.go

```diff
@@ -26,6 +26,8 @@ func startProxyReportErrorIfAny(kubernetesProvider *kubernetes.Provider, cancel
 			"Try setting different port by using --%s", errormessage.FormatError(err), configStructs.GuiPortTapName))
 		cancel()
 	}
+
+	logger.Log.Debugf("done starting proxy")
```

Comment:
it is not "done starting proxy" -> it is more like "proxy ended"

Reply:
but the proxy is not shutting down, it started running at that point